sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/OffsetSeqLog.scala

```diff
@@ -56,7 +56,6 @@ class OffsetSeqLog(sparkSession: SparkSession, path: String)
       throw new IllegalStateException("Incomplete log file")
     }
 
-    val version = parseVersion(lines.next(), OffsetSeqLog.VERSION)
```

Comment:
The problem here is that this removes a call to lines.next(), which has a side effect. I think you can only remove `val version =`. Same above

Reply:
Thanks